schema/pgsql/schema.sql

```diff
@@ -48,55 +48,84 @@ CREATE TABLE channel (
     -- for now type determines the implementation, in the future, this will need a reference to a concrete
     -- implementation to allow multiple implementations of a sms channel for example, probably even user-provided ones
 
+    changed_at bigint NOT NULL DEFAULT EXTRACT(EPOCH FROM NOW()) * 1000,
+    deleted boolenum NOT NULL DEFAULT 'n',
+
     CONSTRAINT pk_channel PRIMARY KEY (id)
 );
 
+CREATE INDEX idx_channel_changed_at ON channel(changed_at);
+
 CREATE TABLE contact (
     id bigserial,
     full_name citext NOT NULL,
-    username citext, -- reference to web user
+    username citext, -- reference web user
```

Comment:
This change doesn't make sense to me. Is this a accidental change?

Reply:
Actually no. This was the result of multiple changes :upside_down_face: